content/blog/arduino-esp-now-lib/index.md

```diff
@@ -0,0 +1,456 @@
+---
+title: "Using ESP-NOW in Arduino"
+date: 2024-06-03T09:00:24-03:00
+tags: ["Arduino", "ESP-NOW", "Wireless", "Wi-Fi", "ESP32", "ESP32-S2", "ESP32-S3", "ESP32-C3", "ESP32-C6"]
+showAuthor: false
+authors:
+  - "lucas-vaz"
+---
+
+ESP-NOW is a connectionless Wi-Fi communication protocol developed by Espressif for its microcontrollers. It allows for efficient, low-power, and low-latency peer-to-peer communication. This article will guide you through the essentials of using the new ESP-NOW library included in Arduino 3.0.0, from basic setup to advanced features.
+
+## What is ESP-NOW?
+
+ESP-NOW enables direct communication between multiple devices without needing a router, ideal for applications requiring real-time data transfer with minimal delay. Key features include:
+- **Low Latency:** Real-time data transfer.
+- **Low Power Consumption:** Suitable for battery-powered devices.
+- **Peer-to-Peer Communication:** Direct device-to-device communication.
+- **Scalability:** Supports multiple peers in a network.
+- **Security:** Built-in encryption for secure data transfer.
+
```

Comment:
Can you add some applications that ESP-NOW could be a good solution?
The ESP-DRONE works with ESP-NOW but not using Arduino core.
https://github.com/espressif/esp-drone